src/Castle.Core/Castle.Core-VS2017.csproj

```diff
@@ -1,42 +1,49 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
-  <Import Project="..\..\buildscripts\common.props"></Import>
```

Comment:
Really messed with this on Visual Studio Code(Ubuntu 16.04LTS) and VS2017(Win10-x64), I think it was crap tabbed before and my system fixed it.